<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>824</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>-8</x>
      <y>-1</y>
      <width>1401</width>
      <height>831</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="myGrafics.qrc">:/newBackground.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="HomeWiget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>1061</width>
      <height>701</height>
     </rect>
    </property>
    <widget class="QLabel" name="HomeImage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1061</width>
       <height>701</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myGrafics.qrc">:/tempLayout.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="LivingroomButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myGrafics.qrc">
       <normaloff>:/termometer.png</normaloff>:/termometer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="UpstairsSmallBedroomButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>480</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myGrafics.qrc">
       <normaloff>:/termometer.png</normaloff>:/termometer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="BathroomButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myGrafics.qrc">
       <normaloff>:/termometer.png</normaloff>:/termometer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="LivingroomTermometer" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLCDNumber" name="TemperatureDisplay">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="DigitalTermometer">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-170</y>
        <width>451</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myGrafics.qrc">:/digitalTermometer.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="TurnoffButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myGrafics.qrc">
        <normaloff>:/powerButton.png</normaloff>:/powerButton.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>DigitalTermometer</zorder>
     <zorder>TemperatureDisplay</zorder>
     <zorder>TurnoffButton</zorder>
    </widget>
    <widget class="QWidget" name="BathroomTermometer" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLCDNumber" name="TemperatureDisplay_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="DigitalTermometer_2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-170</y>
        <width>451</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myGrafics.qrc">:/digitalTermometer.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="TurnoffButton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myGrafics.qrc">
        <normaloff>:/powerButton.png</normaloff>:/powerButton.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>DigitalTermometer_2</zorder>
     <zorder>TemperatureDisplay_2</zorder>
     <zorder>TurnoffButton_2</zorder>
    </widget>
    <widget class="QWidget" name="UpstairsBigBedroomTermometer" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>210</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLCDNumber" name="TemperatureDisplay_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="DigitalTermometer_3">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-170</y>
        <width>451</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myGrafics.qrc">:/digitalTermometer.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="TurnoffButton_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myGrafics.qrc">
        <normaloff>:/powerButton.png</normaloff>:/powerButton.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>DigitalTermometer_3</zorder>
     <zorder>TurnoffButton_3</zorder>
     <zorder>TemperatureDisplay_3</zorder>
    </widget>
    <widget class="QWidget" name="UpstairsSmallBedroomTermometer" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>420</y>
       <width>391</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QLCDNumber" name="TemperatureDisplay_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="DigitalTermometer_4">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>-170</y>
        <width>451</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myGrafics.qrc">:/digitalTermometer.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="TurnoffButton_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myGrafics.qrc">
        <normaloff>:/powerButton.png</normaloff>:/powerButton.png</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>DigitalTermometer_4</zorder>
     <zorder>TemperatureDisplay_4</zorder>
     <zorder>TurnoffButton_4</zorder>
    </widget>
    <widget class="QPushButton" name="UpstairsBigBedroomButton">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>290</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="myGrafics.qrc">
       <normaloff>:/termometer.png</normaloff>:/termometer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ShutDownButton">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myGrafics.qrc">
      <normaloff>:/redX.png</normaloff>:/redX.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="myGrafics.qrc"/>
 </resources>
 <connections/>
</ui>
